tests/pages/firefox/base.py

```diff
@@ -12,8 +12,11 @@
 class ScrollElementIntoView(object):
 
     def scroll_element_into_view(self, strategy, locator):
+        # scroll elements so they are not beneath the header
+        page = self.page if isinstance(self, Region) else self
+        offset = {'x': 0, 'y': -page.family_navigation.root.size['height']}
```

Comment:
Not every Firefox page has the family navigation (most do, but not all). I'm more thinking in future where we might need this function for some other element (even outside the firefox pages).